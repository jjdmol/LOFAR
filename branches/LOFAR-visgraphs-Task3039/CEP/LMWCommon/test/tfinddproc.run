#!/bin/sh

# tfinddproc: Test program finddproc
#
# Copyright (C) 2008
# ASTRON (Netherlands Institute for Radio Astronomy)
# P.O.Box 2, 7990 AA Dwingeloo, The Netherlands
#
# This file is part of the LOFAR software suite.
# The LOFAR software suite is free software: you can redistribute it and/or
# modify it under the terms of the GNU General Public License as published
# by the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# The LOFAR software suite is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along
# with the LOFAR software suite. If not, see <http://www.gnu.org/licenses/>.
#
# @author Ger van Diepen <diepen AT astron nl>
#
# $Id$


../src/finddproc tfinddproc.in_vd tfinddproc.in_cd
echo
../src/finddproc -dirs /usr/local/xyx0,/usr/local/xyx1 tfinddproc.in_cd
echo
../src/finddproc -ndir 1 -dirs /usr/local/xyx0,/usr/local/xyx1 tfinddproc.in_cd
echo
../src/finddproc -ndir 5 -dirs /usr/local/xyx0,/usr/local/xyx1 tfinddproc.in_cd
