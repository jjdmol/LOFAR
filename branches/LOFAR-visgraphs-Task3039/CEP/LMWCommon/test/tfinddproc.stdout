node1#/usr/local/xyx0,node1:/usr,/usr/local/xyx0.vds
node2#/usr/local/xyx1,node1:/usr,/usr/local/xyx1.vds
node1#/usr/local/xyx2,node1:/usr,/usr/local/xyx2.vds

node1#/usr/local/xyx0,node1:/usr,/usr/local/xyx0
node2#/usr/local/xyx1,node1:/usr,/usr/local/xyx1

node1#/usr/local/xyx0,node1:/usr,/usr/local/xyx0

node1#/usr/local/xyx0,node1:/usr,/usr/local/xyx0
node2#/usr/local/xyx1,node1:/usr,/usr/local/xyx1
node1#/usr/local/xyx0,node1:/usr,/usr/local/xyx0
node2#/usr/local/xyx1,node1:/usr,/usr/local/xyx1
node1#/usr/local/xyx0,node1:/usr,/usr/local/xyx0
